src/controls/adaptiveCardDesignerHost/AdaptiveCardDesigner.tsx

```diff
@@ -0,0 +1,229 @@
+import * as monacoLoader from '@monaco-editor/loader';
+import { Versions } from 'adaptivecards';
+import { BindingPreviewMode, CardDesigner, FieldDefinition, GlobalSettings, HostContainer, ToolbarChoicePicker } from 'adaptivecards-designer';
+import * as markdownit from "markdown-it";
+import { Spinner, SpinnerSize } from 'office-ui-fabric-react/lib/Spinner';
+import * as React from 'react';
+import { useEffect, useRef, useState } from 'react';
+import { injectContextProperty } from '../adaptiveCardHost/AdaptiveCardHost.Utilities';
+import { fluentUIDefaultTheme } from '../../common/fluentUIThemes/FluentUIDefaultTheme';
+import { addToolbarButton, addToolbarChoicePicker, addToolboxSnippet, convertNullToEmptyString, hideToolbarElement } from './AdaptiveCardDesigner.Helpers';
+import { applyAdaptiveCardDesignerStyles } from './AdaptiveCardDesigner.Styles';
+import { AdaptiveCardHostContainer, AdaptiveCardHostContainerType } from './fluentUI/AdaptiveCardHostContainer';
+import { initializeDesignerPeers } from './fluentUI/peers/DesignerPeers';
+import { IAdaptiveCardDesignerHostProps } from './IAdaptiveCardDesignerProps';
+
+export const EmptyCard = {
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
```

Comment:
Not a breaking thing but could you please use either `'` or `"` in a single file? :)